Formula/trino.rb

```diff
@@ -0,0 +1,102 @@
+class Trino < Formula
+  desc "Distributed SQL query engine for big data"
+  homepage "https://trino.io"
+  url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/351/trino-server-351.tar.gz"
+  sha256 "e4d50a4179868ee08dd879a50b6c9e2cadf3e7ccce93c6fe9b40f8f7f973e158"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  resource "trino-cli" do
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/351/trino-cli-351-executable.jar"
+    sha256 "f525c91d2872a9bad7cc5bd83131eba30ef48c3e61378858c5a2d8471bc03605"
+  end
+
+  def install
+    libexec.install Dir["*"]
+
+    (libexec/"etc/node.properties").write <<~EOS
```

Comment:
These need to be provided by upstream. We are not going to maintain our own config files.